src/app/containers/ConsentBanner/CanonicalLogic/setCookieOven.js

```diff
@@ -1,11 +1,20 @@
 import cookieOvenUrl from './cookieOvenUrl';
 import 'isomorphic-fetch';
 
-const cookieOvenFetch = async (cookie, value, logger) => {
+const cookieOvenFetch = async (policy, logger) => {
   if (window.location && window.location.origin) {
     try {
       await fetch(
```

Comment:
@avengerpenguin can you explain why we now need to hit .com and .co.uk? Thanks

Reply:
We set the policy entirely by HTTP because cookies set by JS can be deleted after a week in Safari. I think https://webkit.org/blog/8828/intelligent-tracking-prevention-2-2/ is the relevant article on that.

Reply:
ah ok thanks!